docker/README.md

```diff
@@ -0,0 +1,11 @@
+# AMICI with Docker
+
+## Create image
+
+In the AMICI base directory run:
+
+```bash
+git archive -o docker/amici.tar.gz --format=tar.gz HEAD
```

Comment:
add this as CI test?

Reply:
Planning to set up deployment to dockerhub. Would do that soonish, but separately.

Reply:
Done.